sdk/src/wallet/account/operations/balance.rs

```diff
@@ -19,6 +17,12 @@ impl AccountHandle {
     /// Get the AccountBalance
     pub async fn balance(&self) -> crate::wallet::Result<AccountBalance> {
         log::debug!("[BALANCE] get balance");
+        let mut account_balance = AccountBalance::default();
+        #[cfg(feature = "participation")]
+        {
+            account_balance.base_coin.voting_power = self.get_voting_power().await?;
```

Comment:
Does this need a changelog entry?

Reply:
Not really, was already done here https://github.com/iotaledger/iota-sdk/pull/243/files#diff-cac9231564f0d596c25a156c4d2020ae6aba9c03e27eb430e78c0d2c32ecec2fL294, just been re-arranged